### ISSUE TITLE
[Question] Bank editor/MIDI channel -> chip assignment tool?

### ISSUE THREAD

**Issue by freq-mod** [Wohlstand/libEDMIDI#6]
I don't know much about the scope of this project. Emu de MIDI seems to be some kind of MIDI synth emulating AY-PSG, Konami SCC wavetable synth and Yamaha OPLL FM synth. Sound bank seems hardcoded, and many timbres sound _not so great_. If there was a way to alter it, change SCC waves, PSG envelopes, maybe allow instrument to be played with only one sound sources (most layer PSG and FM), it would get pretty cool

**Comment by Wohlstand** [comment]
First off, it needs to research of the bank data, and craft the bank format which is a required thing, but idea sounds cool to allow bank customization here :fox_face:

**Comment by freq-mod** [comment]
Yeah, just for a future. Bank format would have to resesemble WOPN v3, bc multiple sound sources (PSG, SCC, OPLL FM).

As for bank data, only thing I know is `SccWave.h` seems to contain 34 hardcoded SCC waves.
